content/en/docs/distributions/gke/customizing-gke.md

```diff
@@ -156,78 +157,131 @@ options:
+```bash
+kubectl --context=${MGMTCTXT} --namespace=${KF_PROJECT} apply -f <path-to-gpu-nodepool-file>
+```
 
 After adding GPU nodes to your cluster, you need to install NVIDIA's device drivers to the nodes. Google provides a DaemonSet that automatically installs the drivers for you.
 To deploy the installation DaemonSet, run the following command:
+
+```bash
+kubectl --context=${KF_NAME} apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/container-engine-accelerators/master/nvidia-driver-installer/cos/daemonset-preloaded.yaml
 ```
-kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/container-engine-accelerators/master/nvidia-driver-installer/cos/daemonset-preloaded.yaml
+
+
+To disable node-autoprovisioning, edit content for `${KF_DIR}/common/cluster/upstream/cluster.yaml`, set 
+[`enabled`](https://github.com/kubeflow/manifests/blob/4d2939d6c1a5fd862610382fde130cad33bfef75/gcp/deployment_manager_configs/cluster-kubeflow.yaml#L73) 
+to `false`:
+
+```
+    ...
+    clusterAutoscaling:
+      enabled: false
+      autoProvisioningDefaults:
+    ...
 ```
 
 
 ### Add Cloud TPUs to your cluster
 
 Set [`enableTpu:true`](https://cloud.google.com/config-connector/docs/reference/resource-docs/container/containercluster)
-in `${KF_DIR}/common/cluster/upstream/cluster.yaml`:
+in `${KF_DIR}/common/cluster/upstream/cluster.yaml` and enable alias IP (VPC-native traffic routing):
 
 ```
+apiVersion: container.cnrm.cloud.google.com/v1beta1
+kind: ContainerCluster
+...
+spec:
   ...
   enableTpu: true
+  networkingMode: VPC_NATIVE
+  networkRef:
+    name: containercluster-dep-vpcnative
+  subnetworkRef:
+    name: containercluster-dep-vpcnative
+  ipAllocationPolicy:
+    servicesSecondaryRangeName: servicesrange
+    clusterSecondaryRangeName: clusterrange
   ...
+...
+---
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeNetwork
+metadata:
+  name: containercluster-dep-vpcnative
+spec:
+  routingMode: REGIONAL
+  autoCreateSubnetworks: false
+---
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeSubnetwork
+metadata:
+  name: containercluster-dep-vpcnative
+spec:
+  ipCidrRange: 10.2.0.0/16
+  region: us-west1
+  networkRef:
+    name: containercluster-dep-vpcnative
+  secondaryIpRange:
+  - rangeName: servicesrange
+    ipCidrRange: 10.3.0.0/16
+  - rangeName: clusterrange
+    ipCidrRange: 10.4.0.0/16
+
 ```
 
+You can learn more at [Creating a new cluster with Cloud TPU support](https://cloud.google.com/tpu/docs/kubernetes-engine-setup#new-cluster). And view an example for [Vpc Native Container Cluster](https://cloud.google.com/config-connector/docs/reference/resource-docs/container/containercluster) config connector yaml file.
+
+Note that this field is immutable once cluster is created. You need to create new cluster if existing cluster doesn't have TPU enabled.
```

Comment:
I would recommend that you move this information up to the top of this section. Also, the term cluster is used in the following ways: **GKE clusters** or **Kubeflow Pipelines clusters**. It may be helpful to specify which type of cluster this refers to.